README.md

```diff
@@ -1,70 +1,177 @@
+- Basic knowledge of HTML, CSS, JS (JavaScript) and webpack.
+- How to work with Git and GitHub.
+- Basic knowledge of APIs.
 
-### Making a Progressive Web App
+### Setup
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+Clone this repository to your desired folder:
 
-### Advanced Configuration
+```sh
+  cd my-folder
+  git clone https://github.com/Surafels/Metric-webapp.git
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+### Install
 
-### Deployment
+Install this project with:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+```sh
+  cd Metric-webapp
+  npm install
+```
 
-### `npm run build` fails to minify
+### Usage
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+To run the project run :
+
+    npm start
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npx jest
+```
+
+### Live Demo
+
+ [Live demo link](https://company-stock-data.onrender.com/)
+
+ ### Video 
+
+ [Video link](https://www.loom.com/share/38f70a7ca02a480791a273b7555a60ec?sid=85bcf094-e1e0-4b0c-919e-206446e4ae65)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Surafel Samson**
+
+- GitHub: [@githubhandle](https://github.com/Surafels)
+- Twitter: [@twitterhandle](https://twitter.com/surafelsamson2)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/surafel-samson-4b2635267/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Dark Mode**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Surafels/Metric-webapp/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star and follow the authers on github linkedIn and Twitter
+, you are also welcome for any suggestions🙏
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Nelson Sakwa on Behance and the entire Microverse community and family for the support and motivation.
```

Comment:
- [ ] Great work showing acknowledgment to nelson sakwa. However, kindly link his name to his Behance profile. This will greatly improve your acknowledgment. ⬇️ 
[Nelson Sakwa](https://www.behance.net/sakwadesignstudio)[Nelson Sakwa](https://www.behance.net/sakwadesignstudio)